orm/main.go

```diff
@@ -82,6 +116,16 @@ func createOrmServer() *martini.ClassicMartini {
 		}
 		writeChallenge(rs, challenge)
 	})
+
+	m.Delete("/challenge/:id", binding.MultipartForm(ChallengeForm{}), func(rs http.ResponseWriter, rq *http.Request, f ChallengeForm, params martini.Params) {
```

Comment:
This is already starting to get cumbersome on the main function. It now declares 4 functions inside of it. What about declaring them somewhere else and mentioning them over here?

```golang
m.Delete("/challenge/:id", binding.MultipartForm(ChallengeForm{}), controller.DeleteChallenge)
```